great_expectations/rule_based_profiler/parameter_builder/value_set_multi_batch_parameter_builder.py

```diff
@@ -5,14 +5,13 @@
 from great_expectations.rule_based_profiler.parameter_builder import (
     MetricMultiBatchParameterBuilder,
 )
-from great_expectations.rule_based_profiler.types import (
-    Domain,
-    ParameterContainer,
-    get_parameter_value_by_fully_qualified_parameter_name,
-)
+from great_expectations.rule_based_profiler.types import Domain, ParameterContainer
 from great_expectations.rule_based_profiler.types.parameter_container import (
     ParameterNode,
 )
+from great_expectations.rule_based_profiler.util import (
+    get_parameter_value_and_validate_return_type,
+)
 
 
 class ValueSetMultiBatchParameterBuilder(MetricMultiBatchParameterBuilder):
```

Comment:
As discussed, please add a note that this now filters null values @alexsherstinsky

Reply:
@anthonyburdi Done (in next commit).